avm/ptn/deployment-script/import-image-to-acr/tests/e2e/max/dependencies.bicep

```diff
@@ -0,0 +1,160 @@
+@description('Optional. The location to deploy to.')
+param location string = resourceGroup().location
+
+@description('Required. The name of the Virtual Network to create.')
+param virtualNetworkName string
+
+@description('Required. The name of the Managed Identity to create.')
+param managedIdentityName string
+
+@description('Required. The name of the Azure Container Registry to create.')
+param acrName string
+
+@description('Required. The name of the Storage Account to create.')
+param storageAccountName string
+
+var registryRbacRoles = ['7f951dda-4ed3-4680-a7ca-43fe172d538d', '8311e382-0749-4cb8-b61a-304f252e45ec'] // ArcPull, AcrPush
+var storageAccountRbacRole = '69566ab7-960f-475b-8e7c-b3118f30c6bd' // Storage File Data Privileged Contributor
```

Comment:
As only used in one location it may be easier to understand the solution if the GUIDs are directly coded into where they are used. Thoughts?